vomsimporter.py

```diff
@@ -11,13 +11,23 @@
 from VOMSAdmin.VOMSCommands import VOMSAdminProxy
 
 os.environ['SSL_CERT_DIR'] = '/etc/grid-security/certificates'
-DN_CONVERTER_COMMAND = "dn_converter"
 
 
 def convert_dn_rfc2253(dn):
-    rfc_dn = subprocess.check_output(
-        [DN_CONVERTER_COMMAND, dn]).replace("\n", "")
-    return rfc_dn
+    # NOTE: just very naieve / incomplete implementation of DN parsing
+    # https://datatracker.ietf.org/doc/html/rfc4514#section-3
+    sep = re.compile(b'^/(([a-z]+|[0-9](\.[0-9])*)=.*?)(|/[a-z]+=.*)$', re.IGNORECASE)
```

Comment:
```suggestion
    sep = re.compile(b'^/(([a-z]+|[0-9](\.[0-9])*)=.+?)(|/[a-z]+=.*)$', re.IGNORECASE)
```
Shouldn't we accept at least one character here? maybe also at the end.